src/VideoToolbox/VTDefs.cs

```diff
@@ -246,4 +250,28 @@ public enum VTDownsamplingMode {
 		Decimate,
 		Average
 	}
+
+	[Mac (10,13), iOS (11,0), TV (11,0)]
+	public struct VTInt32Point {
```

Comment:
These two structures look useless, they're not used anywhere else from what I can see. Maybe just remove them?

Reply:
Yeah good point, they seem to be unused